### PR TITLE
Request to merge in 1.0.3 changes

### PR DESCRIPTION
**Pull request by fortijames**
Second Pull Request due to conflicts from the first. 

Release of 1.0.3:
 - Add support for an exciting new Script Type: `Custom`
 - Modify ZTP Profile auto-assign to be more efficient and stop rendering device searches once the profile is found.

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -19,20 +19,20 @@
     "help": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/blob/release/1.0.2/README.md",
```
Comment (cs-rohit-khune) [inline info.json]: update version from "1.0.2" to "1.0.3" in help link

File: info.json [hunk 2]
```diff
@@ -217,7 +217,7 @@
         ],
         "recordSets": [
             {
-                "name": "Announcements",
+                "name": null,
```
Comment (cs-rohit-khune) [inline info.json]: update the value of key "name" to "Announcements"

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

File: info.json (hunk 1 above)
Comment (fortijames) [reply info.json]: done

File: info.json (hunk 2 above)
Comment (fortijames) [reply info.json]: done

**Comment by fortijames** [comment]
I updated the info.json. 
Please reach out to me directly if there are any pending actions to merge and release.

**Review by cs-rohit-khune — APPROVED** [review]
Approved

**Pull request merged**